specification/purview/data-plane/Azure.Analytics.Purview.DataMap/preview/2023-10-01-preview/purviewdatamap.json

```diff
@@ -1551,6 +1630,181 @@
+        "description": "Export assets to a given storage.",
+        "operationId": "Entity_Export",
+        "x-ms-examples": {
+          "Entity_Export": {
+            "$ref": "./examples/Entity_Export.json"
+          }
+        },
+        "parameters": [
+          {
+            "in": "body",
+            "name": "exportRequest",
+            "description": "export request",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ExportRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/purviewAPIVersion"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. A job to export entities has been accepted.",
+            "headers": {
+              "Operation-Location": {
+                "description": "Response header for long running operation",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    },
+    "/entity/operations": {
+      "get": {
+        "tags": [
+          "Entity"
+        ],
+        "description": "List entity asynchronous operations.",
+        "operationId": "Entity_ListOperationStatus",
+        "x-ms-examples": {
+          "Entity_ListOperationStatus": {
+            "$ref": "./examples/Entity_ListOperationStatus.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "dayRange",
+            "in": "query",
+            "description": "Return the import operations within the day range. The value will be 7 by default.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "type",
+            "in": "query",
+            "description": "The job type for the asynchronous operation",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/purviewAPIVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Return the status of operations.",
+            "schema": {
+              "type": "array",
+              "description": "An array of operations.",
+              "items": {
+                "$ref": "#/definitions/EntityAsyncOperation"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the request failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/entity/operations/{operationGuid}": {
```

Comment:
This should be `operationId` everywhere even if it is a GUID. Generally, though, customers should be able to provide any value here. GUIDs are generally handy, but should not be required for a resource operation e.g., `PUT`.